src/hammer-vlsi/synthesis/vivado/file_templates/board.tcl

```diff
@@ -0,0 +1,5 @@
+# Vivado fragment to define the FPGA board parameters.
+
+set name {{'{'}}{{ board_name }}{{'}'}}
```

Comment:
I think it makes sense to use jinja2 if the templates we're subsituting are very complex. However in our case if it's just simple substitutions, I think we may be able to just get away with just regular Python string substitution.

Also I would say for fragments this simple, we may as well just embed them into the Python source code.

Reply:
So you prefer to have them embedded in source code instead of having separate template files?

Just asking because for BAG I went the other direction (preferring files + pkg_resources instead of embedding in source) as I found it to be editor friendly; you can open it in an editor and the file format gets recognized, you have syntax highlighting, completions, auto indent, etc., and it's a lot more readable.

I'm fine with either approach, just let me know your decision.

Reply:
I like having separate template files for the reasons you just mentioned.

Reply:
Long-form thoughts below below